src/components/AddBook.jsx

```diff
@@ -1,14 +1,37 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+/* eslint-disable import/extensions */
```

Comment:
- [ ] Kindly remove this line  & solve the linter manually. As this is going to be mandatory in the next milestone.